apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/html/libraries/iframe-edit-mode.js.ts

```diff
@@ -452,6 +453,26 @@ export const EDIT_PAGE_JS = `
     }
 
     // D&D Img - End
+}
+
+    /*
+        This setInterval is required because this script is running
+        before the web component finishes rendering. Currently,
+        we do not have a way to listen to the web component event,
+        so this setInterval is used.
+    */
+
+    let attempts = 0;
+    const initScript = setInterval(function() {
+        const isContainer = document.querySelector('[data-dot-object="container"]');
+        attempts++;
+        if(isContainer) {
```

Comment:
are you sure that only finding 1 container, is enough to execute the `initDragAndDrop` fn for the rest of the containers?